enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -586,7 +586,7 @@ func (c *FirecrackerContainer) LoadSnapshot(ctx context.Context, workspaceDirOve
 
 	var netNS string
 	if c.constants.EnableNetworking {
-		netNS = "/var/run/netns/" + c.id
+		netNS = "/var/run/netns/" + networking.NamespacePath(c.id)
```

Comment:
nit (sorry, I should've been more clear) -- I was thinking `NamespacePath` would return `"/var/run/netns/bb-prefix-" + c.id` -- but this looks good too if you want to just rename it to `Namespace()` or `NamespaceName()`?

Reply:
Ohhh, that's a good idea -- added it and renamed this one to "NetNamespace".